rocAL/source/meta_data/label_reader_folders.cpp

```diff
@@ -151,7 +151,7 @@ void LabelReaderFolders::read_files(const std::string& _path) {
             std::string file_extension = filename.substr(file_extension_idx + 1);
             std::transform(file_extension.begin(), file_extension.end(), file_extension.begin(),
                            [](unsigned char c) { return std::tolower(c); });
-            if ((file_extension != "jpg") && (file_extension != "jpeg") && (file_extension != "png") && (file_extension != "ppm") && (file_extension != "bmp") && (file_extension != "pgm") && (file_extension != "tif") && (file_extension != "tiff") && (file_extension != "webp"))
+            if ((file_extension != "jpg") && (file_extension != "jpeg") && (file_extension != "png") && (file_extension != "ppm") && (file_extension != "bmp") && (file_extension != "pgm") && (file_extension != "tif") && (file_extension != "tiff") && (file_extension != "webp") && (file_extension != "wav"))
```

Comment:
how about other audio files like mp4, aac etc

Reply:
Currently we have added support for wav and tested with the same for RNNT training use case, In order to support mp3 and other audio formats we need to test extensively, hence we will add the support for these in future.